src/components/edit_alarm_definition.js

```diff
@@ -73,13 +73,13 @@ export class EditAlarmDefinitionPageCtrl {
 
     this.monasca.getAlarmDefinition(this.id)
       .then(alarm_definition => {
-	this.savedAlarmDefinition = this.pickKnownFields(alarm_definition);
-	this.newAlarmDefinition = _.cloneDeep(this.savedAlarmDefinition);
+	      this.savedAlarmDefinition = this.pickKnownFields(alarm_definition);
```

Comment:
Please put all re-formatting commits in a separate commit. The idea is that it makes the actual changes to the code easier to pick out.